FAQ.md

```diff
@@ -108,6 +110,13 @@ trying to figure out why `dep` is doing what it does, understanding that its
 basic action is to attempt versions in this order should help you to reason
 about what's going on.
 
+## What external tools are supported?
+During `dep init` configuration from other dependency managers are detected
+and imported, unless `-skip-tools` is specified.
+The following tools are supported: glide, and godep. See 
+[#186](https://github.com/golang/dep/issues/186#issuecomment-306363441) for
+how to add support for another tool.
```

Comment:
Thanks, that looks much better. I was attempting to group this near "How does `dep` decide what version of a dependency to use?" and not break up the "meta" questions on dep.